docs/UserGuide.md

```diff
@@ -180,6 +180,18 @@ Examples:
 * `list -c` followed by `delete -c 2` deletes the 2nd contact in Athena.
 * `find -c n/Betsy` followed by `delete -c 1` deletes the 1st contact in the results of the `find` command.
 
+#### Deleting contacts by Tag : `deletetag`
```

Comment:
Not sure I agree with `deletetag`, sounds misleading and conflicting with the delete inside the tag structure. May want to consider changing to `deletetagged`, or something that is different.

Reply:
agreed, or maybe `deleteByTag` (although that also sounds like a mouthful). the ideal is for us to be able to group it under `delete`.

Reply:
changed to deletebytag